google/cloud/sql/connector/utils.py

```diff
@@ -77,3 +80,27 @@ def write_to_file(
         priv_out.write(priv_key)
 
     return (ca_filename, cert_filename, key_filename)
+
+
+def clean_input(default_ip_type: Any, default_timeout: int, **kwargs: Any) -> Any:
```

Comment:
We're only using this in the 1 place - is it really worth separating out into it's own function?